ONSdigital/eq-survey-runner
# Basic navigation

### Changes

Adds support for basic navigation across multi-page surveys

NOTE: The Url scheme used by the application _HAS CHANGED_

Tests have been refactored, but analytics will need to be updated post-deployment
### How to test

1) Check out the branch
2) Run the unit tests
3) Enable dev mode and choose the 999 survey
4) Navigate through the survey, checking validation and submission.  Check the functionality of the back button
### Who can test

Anyone except @weapdiv-david
